resource/systemd/errors.go

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2016 Asteris, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package systemd
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-multierror"
+)
+
+func MultiErrorAppend(errs ...error) error {
```

Comment:
MultiError has a bug where if you append two nil errors you get an error instead of nil. This will
 rectify that.